tests/html/imports.html

```diff
@@ -12,6 +12,7 @@
   <head>
     <title>Custom Elements: imports integration</title>
     <script>
+      window.customElements = window.customElements || {};
```

Comment:
If we do that, then `forcePolyfill` doesn't get added in the case that `customElements` already exists (native).